MaxText/layers/models.py

```diff
@@ -160,6 +163,18 @@ def get_decoder_layer(self):
     elif self.config.model_name.startswith("gamma"):
       from layers import gamma
       return gamma.GammaDecoderLayer
+    elif self.config.model_name.startswith("gpt3"):
```

Comment:
This code has changed a little but you'll easily fix this on rebase.

Reply:
Sure.